NeosModLoader/NeosModLoader.csproj

```diff
@@ -24,18 +28,23 @@
 
   <ItemGroup>
     <PackageReference Include="Lib.Harmony" Version="2.2.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" Condition="'$(Headless)'=='true'" />
+    <Reference Include="Newtonsoft.Json" Condition="'$(Headless)'=='false'">
+      <HintPath>$(NeosPath)Neos_Data\Managed\Newtonsoft.Json.dll</HintPath>
+    </Reference>
     <Reference Include="BaseX">
       <HintPath>$(NeosPath)Neos_Data\Managed\BaseX.dll</HintPath>
     </Reference>
     <Reference Include="FrooxEngine">
       <HintPath>$(NeosPath)Neos_Data\Managed\FrooxEngine.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>$(NeosPath)Neos_Data\Managed\Newtonsoft.Json.dll</HintPath>
-    </Reference>
   </ItemGroup>
 
   <Target Name="PostBuild" AfterTargets="PostBuildEvent" Condition="'$(CopyToLibraries)'=='true'">
     <Copy SourceFiles="$(TargetPath)" DestinationFolder="$(NeosPath)Libraries" />
+    <Message Text="Copied $(TargetFileName) to $(NeosPath)Libraries" Importance="high" />
   </Target>
+
 </Project>
+
+

```

Comment:
Nitpick, multiple extra unnecessary empty lines

Reply:
Fixed in 66c788df